exercises/perfect-numbers/divisors.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var Divisors = function() {
+
+}
```

Comment:
Sure, I will add the semicolons! Sorry, I didn't payed attention to them in the rest of the code...